src/h5web/vis-packs/core/utils.ts

```diff
@@ -188,12 +188,7 @@ export function getIntegerTicks(domain: Domain, count: number): number[] {
   const stop = Math.floor(max / step);
   const numTicks = stop - start + 1;
 
-  const ticks = new Array(numTicks);
-  for (let i = 0; i < numTicks; i++) {
-    ticks[i] = (start + i) * step;
-  }
-
-  return ticks;
+  return Array.from({ length: numTicks }, (_, i) => (start + i) * step);
```

Comment:
Didn't know this syntax for array intialisation existed. According to https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-new-array.md, it's clearer than `new Array(n)` because `n` can't be confused with an element of the array.

I especially like the second parameter of `Array.from` -- it's feels nicer than chaining the constructor call with `fill` and `map` like we do in other places.